main.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const { createServer } = require('./src/createServer');
+
+createServer()
+  .listen(3000, () => {
```

Comment:
since you're already using `.env` you could pass port to `.listen` method, it is popular practice 😄